### PR TITLE
Fixing Dialog width issue and removing extra ms-Dialog div

### PR DESCRIPTION
**Pull request by atneik**

### CODE REVIEW

File: src/components/Dialog/Dialog.tsx [hunk 1]
```diff
@@ -65,7 +65,6 @@ export class Dialog extends React.Component<IDialogProps, any> {
             ariaDescribedBy={ subText ? id + '-subText' : '' }
             onDismiss={ onDismiss }
             >
-            <div className={ dialogClassName }>
```
Comment (aygarg) [inline src/components/Dialog/Dialog.tsx]: Where will you apply the dialogClassName? We need this to be able to override styles.

Comment (atneik) [inline src/components/Dialog/Dialog.tsx]: This is was the cause that we saw two ms-Dialog. It is already there at: https://github.com/OfficeDev/office-ui-fabric-react/blob/master/src/components/Dialog/Dialog.tsx#L62

Comment (dzearing) [inline src/components/Dialog/Dialog.tsx]: I'm confused; the issue was setting maxwidth:

$Dialog-default-max-width: 340px;

File: src/components/Dialog/Dialog.scss [hunk 2]
```diff
@@ -83,24 +83,23 @@ $Dialog-default-max-width: 340px;
...
 }
 
 .ms-Dialog-title {
   margin: 0;
-  display: table-cell;
   @include ms-font-xl;
   color: $ms-color-neutralPrimary;
-  padding: 20px 28px 20px;
-  width: 100%;
+  @include padding(20px, 36px, 20px, 28px);
```
Comment (dzearing) [inline src/components/Dialog/Dialog.scss]: This is asymmetrical. Possible to remove that? I think maybe it's to leave room for a close button or something...

**Pull request merged**